modules/sriov-operator-metrics.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="sriov-operator-metrics_{context}"]
+= Enabling the SR-IOV network metrics exporter
+
+The Single Root I/O Virtualization (SR-IOV) network metrics exporter is disabled by default. To enable the metrics exporter, you must set the `spec.featureGates.metricsExporter` field to `true`.
+
+[IMPORTANT]
+====
+When the metrics exporter is enabled, the SR-IOV Network Operator deploys the metrics exporter only on nodes with SR-IOV capabilities.
+====
+
+.Prerequisites
+
+* You have installed the {oc-first}.
+* You have logged in as a user with `cluster-admin` privileges.
+* You have installed the SR-IOV Network Operator.
+
+.Procedure
+
+. Enable cluster monitoring by running the following command:
++
+[source,terminal]
+----
+$ oc label ns/openshift-sriov-network-operator openshift.io/cluster-monitoring=true
+----
++
+To enable cluster monitoring, you must add the `openshift.io/cluster-monitoring=true` label in the namespace where you have installed the SR-IOV Network Operator.
+
+. Set the `spec.featureGates.metricsExporter` field to `true` by running the following command:
++
+[source,terminal]
+----
+$ oc patch -n openshift-sriov-network-operator sriovoperatorconfig/default \
+    --type='merge' -p='{"spec": {"featureGates": {"metricsExporter": true}}}'
+----
++
+[TIP]
+====
+You can alternatively apply the following YAML to update the Operator:
+
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovOperatorConfig
+metadata:
+  name: default
+  namespace: openshift-sriov-network-operator
+spec:
+  featureGates:
+    metricsExporter: true
+----
+====
```

Comment:
We should avoid showing this type of alternative method unless it represents a substantially different use case. From what I understand, `oc patch` is preferred to manually editing YAML config files.
```suggestion
```

Reply:
Updated as per the suggestion.